lib/ansible/modules/net_tools/cloudflare_dns.py

```diff
@@ -42,9 +42,8 @@
     required: false
     default: "1"
   proto:
-    description: Service protocol. Required for C(type=SRV)
+    description: Service protocol. Required for C(type=SRV). Common values are tcp and udp. (Before Ansible 2.6 only tcp and udp were available)
```

Comment:
Also, add this note is [CHANGELOG.md](https://github.com/ansible/ansible/blob/devel/CHANGELOG.md#minor-changes) telling end-user about this.